test/data/biosamples_to_sites.json

```diff
@@ -91,7 +91,7 @@
       }
     },
     {
-      "id": "nmdc:biosample_placeholder_4",
+      "id": "nmdc:bsm-biosample_placeholder_4",
       "name": "Root microbial communities from poplar common garden site in Clatskanie, Oregon, USA - BESC-13-CL2_39_29 endosphere",
       "description": "Root microbial communities from poplar common garden site in Clatskanie, Oregon, USA",
       "collected_from": "GOLD:BESC-13-CL2_39_29",
```

Comment:
This placeholder didn't get updated. Is that intentional? A test that is expected to fail perhaps?

Reply:
No, it's the same un-anchored right pattern that you observed in some of the other `Biosample` test instances.